src/tiledb/cloud/utilities/consolidate.py

```diff
@@ -0,0 +1,239 @@
+from collections import defaultdict
+from os.path import basename
+from typing import Any, Mapping, Optional, Sequence
+
+import tiledb
+from tiledb.cloud import dag
+from tiledb.cloud.rest_api.models import RetryStrategy
+from tiledb.cloud.utilities import get_logger
+from tiledb.cloud.utilities import max_memory_usage
+from tiledb.cloud.utilities import run_dag
+
+MAX_FRAGMENT_SIZE = 1 << 30  # bytes
```

Comment:
I would just add `_BYTES` to the end of this so that the unit is always right there in the name.